fix(userinfo): hide autoLoginFrame & noPwdLoginFrame

cetc32-ck opened the pull request:
Description: hide autoLoginFrame & noPwdLoginFrame

Log: 大商所项目隐藏自动登录和免密登录
Task: https://172.17.50.104/biz/story-view-11456.html

handsome-feng commented on po/tr.po:
```diff
@@ -517,11 +517,11 @@ msgstr "Kapat"
 #: panels/power/kpm-prefs-core.c:273 panels/power/kpm-prefs-core.c:276
 msgid "Sleep"
-msgstr "Uyut"
+msgstr "Uyku Modu"
```
Should be "Uyku"?

handsome-feng commented on po/tr.po:
```diff
@@ -604,7 +604,7 @@ msgstr "%s şarjı bekliyor (%.1f%%)"
 #: panels/power/kpm-upower.c:410
 msgid "Product:"
-msgstr "Ürün："
+msgstr "Ürün"
```
Why delete the semicolon？

handsome-feng commented:
Thanks and close this since it changes a lot after you submit the PR.

cetc32-ck committed:
fix(userinfo): hide autoLoginFrame & noPwdLoginFrame

Description: hide autoLoginFrame & noPwdLoginFrame

Log: 大商所项目隐藏自动登录和免密登录
Task: https://172.17.50.104/biz/story-view-11456.html
